R/reportWaterUsage.R

```diff
@@ -17,19 +17,19 @@
 
 reportWaterUsage<-function(gdx, detail=TRUE) {
   
-  out <- water_usage(gdx,level="regglo",users=NULL,sum=FALSE,digits=3)[,,"agriculture"]
-  getNames(out) <- "Resources|Water|Withdrawal|Agriculture (km3/yr)"
-   
+  x <- water_usage(gdx, level="regglo",users=NULL, digits=3)[,,"agriculture"]
+  getNames(x) <- "Resources|Water|Withdrawal|Agriculture (km3/yr)"
+  
   if (detail==TRUE) {
-    y <- water_usage(gdx,level="regglo",users="kcr",sum=FALSE,digits=3)[,,"agriculture"]
+    y <- water_usage(gdx,level="regglo",users="kcr",sum=TRUE,digits=3)
```

Comment:
Why `sum=TRUE` here? Is there an error coming when running magpie with different realization of `water_demand` module? One realization deals with all sectors and another one only with agricultture. In that case we have to change the `water_usage` function to account for all cases in the module.

Reply:
sum=FALSE, in this case, does not sum over all crops in the crops category, and the same for livestock. It reports water use for every crop and livestock category. Therefore, sum=TRUE, In this case, sums over the respective crop and livestock categories to report total water use.

Reply:
isn't `sum` in the `water_use` function to sum over sectors? At least the documentation says so. You write as if it's over products.